apps/extension/src/ui/domains/Ethereum/useEthTransaction.ts

```diff
@@ -345,7 +346,7 @@ const useGasSettings = ({
           mapMaxPriority.high = minimumMaxPriorityFeePerGas
       }
 
-      if (evmNetworkId && isOpStackEvmNetwork(evmNetworkId)) {
+      if (evmNetworkId && evmNetwork?.l2FeeType?.type === "op-stack") {
```

Comment:
Is there a problem here for OP stack custom networks that are not defined in chaindata?

Reply:
most likely yes